data/registry/collector-receiver-awscloudwatchmetrics.yml

```diff
@@ -0,0 +1,16 @@
+title: CloudWatch Metrics Receiver
+registryType: receiver
+isThirdParty: false
+language: collector
+tags:
+  - awscloudwatchmetrics
+  - receiver
+  - collector
+repo: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awscloudwatchmetricsreceiver
+license: Apache 2.0
+description:
+  Receives Cloudwatch metrics from AWS Cloudwatch via the AWS SDK for Cloudwatch
+  Logs
+authors: OpenTelemetry Authors
+otVersion: latest
+spelling: cSpell:ignore awscloudwatchmetrics
```

Comment:
I'm wondering if in these files we should go with comments instead of an official YAML field, like this (could be moved to the top of the file):

```suggestion
# cSpell:ignore awscloudwatchmetrics
```

WDYT?

Reply:
ah, that's better!!